src/main/resources/pdl/hent-barn.graphql

```diff
@@ -14,6 +14,30 @@ query ($ident: ID!) {
         statsborgerskap {
             land
         }
+         deltBosted {
+                    startdatoForKontrakt
+                    sluttdatoForKontrakt
```

Comment:
Får advarsel i altair, "Feltet deltBosted.sluttdatoForKontrakt is deprecated. Flyttet til adresser rett under"